e2e/cypress/support/commands.js

```diff
@@ -136,6 +143,48 @@ Cypress.Commands.add('deleteMetadata', (metadataTrackName = 'Track 1') => {
   log.end();
 });
 
+// IMPORTANT only works with files that are uncompressed, gem2s fails with compressed files
+// (probably due to file sizes wrong calculation, if we consider this should be fixed
+// the work should probably be done in the forked repo cypress-file-upload)
```

Comment:
Yes, if I used `matrix.mtx.gz` for example gem2s fails while if I gunzip it and then use `matrix.mtx` it works correctly, I believe it could be related to the sizes of gzipped files when loaded on cypress don't really match the real file size.

Here's a ticket for fixing this: [ticket](https://biomage.atlassian.net/browse/BIOMAGE-1435).

I don't think this ticket is very high priority however so I set it to low since it doesn't block us from testing anything but the upload of gzipped files.

Reply:
Agreed on the lfs thing, I'll try it out then.